ci-operator/config/terraform-redhat/terraform-rhcs-rosa/terraform-redhat-terraform-rhcs-rosa-main.yaml

```diff
@@ -0,0 +1,28 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_path: Dockerfile
+  to: clis-img
```

Comment:
what ``clis-img`` means?

Reply:
It is the name for the image containing the necessary clients for future tests.

Reply:
let's rename to either one of ``terraform-rhcs-rosa-clients``, ``terraform-rhcs-rosa-build``, ``clients``
naming is hard, but it's just too much to use both acronym pluralized and an abbreviation, and it interferes with reading the code